test/expand.coffee

```diff
@@ -145,3 +145,18 @@ testExpand
 				$expand: 'c'
 				$select: [ 'd', 'e' ]
 	'a($expand=b($expand=c&$select=d,e))'
+
+
+
+testExpand(
+	'a/$count': $filter: b: 'c'
+	"a/$count($filter=b eq 'c')"
+)
+
+testExpand
+	a:
+		$expand:
+			b:
+				$expand: 'c/$count'
+				$filter: d: 'e'
+	"a($expand=b($expand=c/$count&$filter=d eq 'e'))"
```

Comment:
Would it also make sense to have a simple test case for
* `a: $expand: 'b/$count': {}`
* `a: $expand: ['b/$count', c: {}]`
?

Reply:
Added some more tests